ParallelRoadTool/ModInfo.cs

```diff
@@ -39,6 +47,35 @@ public void OnSettingsUI(UIHelperBase helper)
         {
             try
             {
+                // BUG: No translated strings in Option screen on Main Menu.
```

Comment:
This block is the exact duplicate of [ParallelRoadTool\ParallelRoadTool.cs:L271-299](https://github.com/ST-Apps/CS-ParallelRoadTool/blob/f4987b2ef247ae1d8f2f35f9cd52166dbb206dda/ParallelRoadTool/ParallelRoadTool.cs#L271-L299), as reported by [CodeFactor.io](https://www.codefactor.io/repository/github/st-apps/cs-parallelroadtool/pull/49).

It should be safe to change [this line](https://github.com/ST-Apps/CS-ParallelRoadTool/blob/f4987b2ef247ae1d8f2f35f9cd52166dbb206dda/ParallelRoadTool/ParallelRoadTool.cs#L267
) to be `public static`, so that you call that method instead of doing copy & paste.

Reply:
Is it OK if I change that method, 'private -> public'?

Reply:
Done. :)

Reply:
I created a new PR, 'fix 2 bugs #52'

Reply:
I recreated a new PR, 'Fixed 2 bugs #53' instead of this.